docs/rules.md

```diff
@@ -0,0 +1,78 @@
+# Rules
```

Comment:
This file was deleted and should be removed.

Reply:
ping

Reply:
done